frontend/src/components/ProductForm/ProductForm.js

```diff
@@ -72,12 +92,20 @@ const ProductForm = ({ match, history }) => {
     setProduct({ ...product, [event.target.name]: event.target.files || event.target.value });
   };
 
+  const editHeader = <h1>{t('first-product:edit')}</h1>
+  const addHeader =
+  <>
+    <h1>{t('first-product:head')}</h1>
+    <p>{t('first-product:text')}</p>
+  </>
```

Comment:
I think we'll have to create a View for FirstProduct again and put this in there and then just load the ProductForm into it. Then also remove the Viewwrapper.View from this file.